### PR TITLE
Convert Event Countdown and Timeline blocks to use SCSS build

### PR DESCRIPTION
**Pull request by mkaz**
#### Changes proposed in this Pull Request

Moves the CSS for the Event Countdown and Timeline blocks to use the build setup with FSE plugin. Also simplified the PHP a little to match other plugins.

#### Testing instructions

* No feature or functionality changes
* Confirm after build the blocks show as expected in editor and front-end

### CODE REVIEW

File: apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+@keyframes gravity {
+	to {
+		transform: translateY( 200px );
+		-webkit-transform: translateY( 200px );
+		opacity: 0;
+	}
+}
+
+@-webkit-keyframes position {
```
Comment (simison) [inline apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss:211]: [Autoprefixer](https://github.com/postcss/autoprefixer) in CSS pipeline likely already adds vendor prefixes when needed, ok to remove them.

File: apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+		margin-left: 20%;
+	}
+
+	80%, 99.9% {
+		margin-top: 30%;
+		margin-left: 80%;
+	}
+}
+
+	@keyframes position {
```
Comment (simison) [inline apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss:238]: Keyframes end up in global CSS scope, hence colliding with anything with same name and this is pretty generic. I'd suggest prefixing these with `jetpack_`, `event_countdown_`, or similar.

Comment (mkaz) [reply apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss:238]: Thanks, I addressed in this PR: https://github.com/Automattic/wp-calypso/pull/40926

File: apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss (hunk 1 above)
Comment (mkaz) [reply apps/full-site-editing/full-site-editing-plugin/event-countdown-block/blocks/src/style.scss:211]: :+1: Removed in https://github.com/Automattic/wp-calypso/pull/40926

**Pull request merged**